specification/containerregistry/data-plane/Microsoft.ContainerRegistry/preview/2019-08-15/containerregistry.json

```diff
@@ -2061,6 +2062,38 @@
         "writeEnabled": true
       }
     },
+    "RepositoryChangeableAttributes": {
+      "description": "Changeable attributes for Repository",
+      "properties": {
+        "deleteEnabled": {
+          "type": "boolean",
+          "description": "Delete enabled"
+        },
+        "writeEnabled": {
+          "type": "boolean",
+          "description": "Write enabled"
+        },
+        "listEnabled": {
+          "type": "boolean",
+          "description": "List enabled"
+        },
+        "readEnabled": {
+          "type": "boolean",
+          "description": "Read enabled"
+        },
+        "teleportEnabled": {
+          "type": "boolean",
+          "description": "Teleport enabled"
```

Comment:
Can you please more description as to what teleport enabled means? read/write/delete are common verbs that we know but for teleport more description will be helpful

Reply:
I am not sure how much description is okay here. I could also link http://aka.ms/acr/teleport.  We also haven't provided any description for QuarantineState or such properties..

Reply:
I understand that quarantine and other properties are not descriptive, but wanted to sure we improve on things as we move forward with new changes.